extensions/wikia/GoogleTagManager/GoogleTagManagerHooks.class.php

```diff
@@ -11,7 +11,7 @@ static public function onWikiaSkinTopScripts( &$vars, &$scripts ) {
 	
 	public static function onBeforePageDisplay( OutputPage $out ) {
 		wfProfileIn( __METHOD__ );
-		global $wgEnableGoogleTagManager;
+		global $wgEnableGoogleTagManagerExt;
```

Comment:
Do we need this variable here?

Reply:
Don't need it. You should done my other pr. 😄